src/custom-lists/reducer.js

```diff
@@ -0,0 +1,172 @@
+import { createReducer } from 'redux-act'
+import union from 'lodash/fp/union'
+
+import * as actions from './actions'
+
+const defaultState = {
+    listCount: null,
```

Comment:
I don't think the `listCount` state is used anywhere, apart from the unused `getListCount` selector.

I'm also not entirely sure if the `urlDragged` state is being used anywhere. I can see corresponding actions being used to set it and where it is mapped to the Overview container's props, however I can't see it being used anywhere after that. Let me know if I'm missing it.

I would also recommend setting the `lists` state type using proper JSDoc typings using `@typedef {Object} ListState` (or just use TS). There's some examples of `@typedef`s in the rest of the codebase, but don't spend too much time on it if it's painful to get right.